spec/pycall/pyobject_wrapper_spec.rb

```diff
@@ -161,7 +161,7 @@ module PyCall
           list = PyCall::List.new([*1..10])
           expect(list[(1..-1).step(2)]).to eq(PyCall::List.new([2, 4, 6, 8, 10]))
           expect(list[(1..-2).step(2)]).to eq(PyCall::List.new([2, 4, 6, 8]))
-          expect(list[(10..1).step(-1)]).to eq(PyCall::List.new([*1..10].reverse))
+          expect(list[(-1..0).step(-1)]).to eq(PyCall::List.new([*1..10].reverse))
```

Comment:
This makes lines 164 and 165 identical. Was that intended? Why not limit the original test to RUBY_VERSION < "3.4" with `expect(list[(nil..nil).step(-1)]).to eq(PyCall::List.new([*1..10].reverse)) if RUBY_VERSION < "3.4"`?

Reply:
It makes sense.